Commands/Commands/utils.js

```diff
@@ -79,7 +78,7 @@ commands.stats = {
           inline: true
         }
       )
-      msg.channel.sendMessage('', false, {
+      msg.channel.openDM('', false, {

```

Comment:
`openDM()` returns a channel as a promise resolve, the way you're trying to do it now won't work.